datadog_checks_dev/datadog_checks/dev/tooling/commands/release.py

```diff
@@ -101,6 +103,26 @@ def create_trello_card(client, teams, pr_title, pr_url, pr_body):
                 break
 
 
+def pick_card_members(config, author, team):
+    """Return a list of members to assign to the created card.
+
+    In practice it returns one trello users which is not the PR author, for the given team.
+    For it to work, you a `trello_user_$team` table in your ddev configuration,
+    with keys being github users and values being their corresponding trello
+    IDs (not names).
+
+    For example::
+        [trello_users_integrations]
+        john = "xxxxxxxxxxxxxxxxxxxxx"
+        alice = "yyyyyyyyyyyyyyyyyyyy"
+    """
+    users = config.get('trello_users_{}'.format(team.lower()))
+    if not users:
+        return
+    member = random.choice([key for user, key in users.items() if user != author])
```

Comment:
Would it be possible to also exclude reviewers?

Reply:
Not easily. This is not in the basic PR data that we fetch.